packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/timing/TimingModuleTest.kt

```diff
@@ -0,0 +1,265 @@
+package com.facebook.react.modules.timing
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.CatalystInstance
+import com.facebook.react.bridge.JavaOnlyArray
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.common.SystemClock
+import com.facebook.react.devsupport.interfaces.DevSupportManager
+import com.facebook.react.modules.core.ChoreographerCompat.FrameCallback
+import com.facebook.react.modules.core.JSTimers
+import com.facebook.react.modules.core.ReactChoreographer
+import com.facebook.react.modules.core.ReactChoreographer.CallbackType
+import com.facebook.react.modules.core.TimingModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.any
+import org.mockito.ArgumentMatchers.eq
+import org.mockito.Mockito.doAnswer
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.reset
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.verifyNoMoreInteractions
+import org.mockito.Mockito.`when` as mockWhen
+import org.mockito.invocation.InvocationOnMock
+import org.mockito.stubbing.Answer
+import org.powermock.api.mockito.PowerMockito
+import org.powermock.core.classloader.annotations.PowerMockIgnore
+import org.powermock.core.classloader.annotations.PrepareForTest
+import org.powermock.modules.junit4.rule.PowerMockRule
+import org.robolectric.RobolectricTestRunner
+
+@[
+PrepareForTest(Arguments::class, SystemClock::class, ReactChoreographer::class)
+PowerMockIgnore("org.mockito.*", "org.robolectric.*", "androidx.*", "android.*")
+RunWith(RobolectricTestRunner::class)]
+class TimingModuleTest {
+  companion object {
+    const val FRAME_TIME_NS = 17 * 1000 * 1000
+  }
+
+  private lateinit var mTimingModule: TimingModule
```

Comment:
Please avoid using Hungarian notation in Kotlin, `timingModule` is a better name in this case (ditto for all the others).